### ISSUE TITLE
Show notification's resume

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#44]
```
Instead of showing tweet by tweet from the timeline after each update Pino 
could show a resume like

"xxxx new tweets"
"xxxx new replys"

```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 13 Jan 2010 at 10:26

**Comment by GoogleCodeExporter** [comment]
```
This is great idea, thanks.
```

Original comment by `tro...@gmail.com` on 14 Jan 2010 at 10:55
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Agreed. It's redundant to display the tweet in notification, especially since 
that
can't be interacted with.
```

Original comment by `benoit.b...@gmail.com` on 17 Jan 2010 at 12:07

**Comment by GoogleCodeExporter** [comment]
```
I only set notifications for replies, so it's useful for me to see the tweet in 
the 
notification and continue with other work. It might be OK to have "XXX new 
replies" 
for multiple tweets, though.
```

Original comment by `eph.zero` on 17 Jan 2010 at 2:06

**Comment by GoogleCodeExporter** [comment]
```
Yeah, i use that feature too, but mainly because to be constantly notified gets 
annoying. With a resume i could also know when Pino updates my timeline. The 
messages 
catch my attention easier than the change in the panel's icon :D
```

Original comment by `mich.ba...@gmail.com` on 18 Jan 2010 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
Closed by 306a61fe0f revision (like option).
```

Original comment by `tro...@gmail.com` on 30 Jan 2010 at 7:02
- Changed state: **Fixed**
